src/modsim2/similarity/metrics/mmd.py

```diff
@@ -0,0 +1,97 @@
+import logging
+
+import numpy as np
+from sklearn import metrics
+
+from modsim2.similarity.embeddings import EMBEDDING_FN_DICT
+
+from .metrics import DistanceMetric
+
+# Set module logger
+logger = logging.getLogger(__name__)
+
+
+class MMD(DistanceMetric):
+    def __init__(self, seed: int):
+        # Kernel dictionary
+        self.__MMD_KERNEL_DICT = {
+            "rbf": metrics.pairwise.rbf_kernel,
+            "laplace": metrics.pairwise.laplacian_kernel,
+        }
```

Comment:
Missing `super().__init__(seed)` call

Reply:
I've added this in though thought the mmd calculations were deterministic

Reply:
Oh true! My bad!